packages/react/src/components/CardEditor/CardEditForm/_card-edit-form.scss

```diff
@@ -173,6 +173,9 @@
       .#{$prefix}--number input[type='number'] {
         padding-right: 0;
       }
+      .#{$prefix}--form-item input[type='text'] {
+        min-width: 9.375rem;
```

Comment:
nit: is this value "hardcoded" for carbon number input elements or where does it come from?

Reply:
Exactly. This is the hard-coded width in the number elements I used here to make them match.